module/functions/Select-ExceptIn.ps1

```diff
@@ -0,0 +1,105 @@
+# <copyright file="Select-ExceptIn.ps1" company="Endjin Limited">
+# Copyright (c) Endjin Limited. All rights reserved.
+# </copyright>
+
+<#
+.SYNOPSIS
+Compares two Hashtable arrays and returns the hashtable entries from the input that don't exist in the reference array.
+
+.DESCRIPTION
+This is intended to find hashtable-based objects that are missing from the reference array of hashtables.  For
+example, as part of some synchronisation logic.
```

Comment:
Is it worth adding a note that this won't work for nested hashtables (i.e. it only works for properties that can be compared with `Compare-Object`)?